Index: app/src/main/java/nurul/id/graphvertexcolouring/Level17.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/nurul/id/graphvertexcolouring/Level17.java	(date 1524867544068)
+++ app/src/main/java/nurul/id/graphvertexcolouring/Level17.java	(date 1524867544068)
@@ -0,0 +1,13 @@
+package nurul.id.graphvertexcolouring;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class Level17 extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_level17);
+    }
+}
